.github/workflows/pages.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Setup Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.1' # Not needed with a .ruby-version file
+          ruby-version: "3.1" # Not needed with a .ruby-version file
```

Comment:
This change is not related to the commit of reverting the files. Hopefully it will go away after a rebase.